javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -25,6 +25,101 @@ public void shouldNotBeInstantiable() {
         AssertionsExtensions.assertThat(API.class).isNotInstantiable();
     }
 
+    //
+    // Alias should return not null.
+    // More specific test for each aliased class implemented in separate test class
+    //
+
+    @Test
+    public void shouldFunction0ReturnNotNull() {
+        assertThat(function0(() -> null)).isNotNull();
```

Comment:
There exists no such think as function equality (in limited time/mem). Two functions `f1: T -> R` and `f2: T -> R` are equal, if

1. f1(t) is defined if and only if f2(t) is defined for all t &in; T
2. f1(t) is defined => f1(t) == f2(t), for all t &in; T

I think `isNotNull` is the best we can do here.